### PR TITLE
Update Hiring Guide

### PR DESCRIPTION
**Pull request by tsolakoua**
- Added duration of on-site interviews and how participants are chosen. Suggestions were by @mcuadros 
- Update First contact.

### CODE REVIEW

File: talent/hiring_process.md [hunk 1]
```diff
@@ -71,6 +74,7 @@ This document explains the internal hiring process at source{d} and the responsi
 ### Exceptions 
 
 - There are exceptions to the above steps for internship roles. The first step in this case is the Coding Challenges and then a technical Skype interview. Also, we do not do on-site interviews for interns unless one could join us as a full-time engineer.
+- If a developer has applied to multiple roles, it will be determined either in the first interview or in the on-site interview after discussion with the leads of the applied roles, in which team fits better.
```
Comment (mcuadros) [inline talent/hiring_process.md]: I don't get the second phase, after which discussion with the leads?

Comment (tsolakoua) [reply talent/hiring_process.md]: I'll rephrase it to make more sense

**Comment by bzz** [comment]
Really nice update @tsolakoua !
